gems/pending/spec/middleware/events/middleware_event_monitor_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "middleware/events/middleware_event_monitor"
+
+describe MiddlewareEventMonitor do
+  subject do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    auth                 = AuthToken.new(:name     => "test",
+                                         :auth_key => "valid-token",
+                                         :userid   => "jdoe",
+                                         :password => "password")
+    ems                  = FactoryGirl.create(:ems_hawkular,
+                                              :hostname        => 'localhost',
+                                              :port            => 8080,
+                                              :authentications => [auth],
+                                              :zone            => zone)
+    described_class.new(ems)
+  end
+
+  context "#each_batch" do
+    # VCR.eject_cassette
+    # VCR.turn_off!(ignore_cassettes: true)
+
+    VCR.configure do |c|
+      c.default_cassette_options = {
+        :match_requests_on => [:method, VCR.request_matchers.uri_without_params(:startTime)]
+      }
+    end
+
+    it "# invoke each_batch" do
+      # if generating a cassette the polling window is the previous 1 minute
+      VCR.use_cassette(described_class.name.underscore.to_s) do # , :record => :new_episodes) do
```

Comment:
ok, now I got, why the cassette is at the wrong location.
The `MiddlewareEventMonitor` should not be at `gems/pending/middleware` but at `app/models/manageiq/providers/hawkular/middleware_manager/event_catcher`
Then also namespace the class to `ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Monitor``